src/pokemon/pokemon.service.ts

```diff
@@ -0,0 +1,81 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Pokemon } from './entities/pokemon.entity';
+import { Repository, SelectQueryBuilder } from 'typeorm';
```

Comment:
Please do not forget about the correct order.

Reply:
Done